### DISCUSSION TITLE
{NEWBIE}  How do I enter the path for my media (Installed via Docker on a NAS)

### DISCUSSION THREAD

**Discussion by irishtoon** [stashapp/stash#5543] [Q&A]
Hi, apologies in advance for what will probably seem like such a basic question. I installed Stash on my ugreen NAS using docker and the yml file (changed port to 8888). The app is running and can access the settings page without issue but I cannot point to the folder on the NAS where the media files are located. I've tried different versions of the folder names in the library path dialog but it can't find anything and I guess it can only see the 'local' files of the docker install. Would someone be willing to point me in the right direction and provide instructions on how to find the files located on //NAS/Movies?

**Comment by feederbox826** [comment]
If it's in docker, the paths are mapped host:container

Unless you specify it, docker won't be able to see the file path, Try adding

```yml
volumes:
 - //NAS/Movies:/Movies
```

and `//NAS/Movies` will appear as `/Movies` in your container
